hal/tests/pinvalidate/pinvalidate.py

```diff
@@ -56,7 +56,7 @@ def find_target_by_path(target_path):
 
     with open(target_path) as pin_names_file:
         pin_names_file_content = pin_names_file.read()
-    
+
```

Comment:
please remove spaces in separate commit

Reply:
PyCharm has removed them automatically...

Reply:
> 
> 
> please remove spaces in separate commit

Done!